listener/listener.go

```diff
@@ -46,19 +44,21 @@ func configure() {
 
 	eventsReader = kafka.NewReader(eventsConfig)
 
+	traceApi := utils.GetenvOrFail("LOG_LEVEL") == "trace"
+
 	inventoryAddr := utils.GetenvOrFail("INVENTORY_ADDRESS")
 
-	config := inventory.NewConfiguration()
-	config.Debug = true
-	config.BasePath = inventoryAddr + INVENTORY_API_PREFIX
+	invConfig := inventory.NewConfiguration()
```

Comment:
Yes, these names are better. But I wouldn't use shortcut here, but name it simply `inventoryConfig`.

Reply:
Fixed.